course/style.css

```diff
@@ -448,6 +448,13 @@ body.page-template-page-wide-width footer .wp-block-group {
 /*
 * Sensei-specific styles
 */
+body[class*="-template-single-course"] .wp-block-post-title,
```

Comment:
I see that the following 2 CSS classes are the ones that style courses and lessons, so I'm wondering which use case this selector is needed for?